File: 2.0/js/media/getMediaDataByKey.js
'strict'

const CallfireClient = require('callfire-api-client-js');
const client = new CallfireClient('api-login', 'api-password');

client.ready(() => {
    client.media.getMediaDataByKey({
      key: 'f733daab-fc05-470d-abe6-87cd136599ad',
      extension: 'mp3'
    })
      .then((response) => {
        // returns binary response
        console.log(response.data);
      })
      .catch((err) => {
        console.log('request error ' + err.data);
      });
  },
  (clientError) => {
    console.log('client error ' + clientError);
  }
);

File: 2.0/python/media/getMediaDataByKey.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.media.getMediaDataByKey(
    key='f733daab-fc05-470d-abe6-87cd136599ad',
    extension='mp3'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)